commit 5eebb7f9288b7881ffb929b1fd494fe3ac3be27d
Author: Russell Senior <russell@personaltelco.net>
Date:   Wed Mar 6 12:49:42 2013 -0800

    bridge-utils: Fix compile against linux-3.8.x
    
    Linux 3.8 has a header, include/uapi/linux/if_bridge.h that uses a
    struct in6_addr but doesn't define it.  The trivial seeming fix of
    including the header that does define it causes more problems.  The
    problem was discussed on mailing lists in January 2013.  The final
    suggestion I found was here:
    
          http://www.redhat.com/archives/libvir-list/2013-January/msg01253.html
    
    This is intended to implement that suggestion.
    
    Signed-off-by: Russell Senior <russell@personaltelco.net>

diff --git a/libbridge/libbridge.h b/libbridge/libbridge.h
index 39964f2..dd14bae 100644
--- a/libbridge/libbridge.h
+++ b/libbridge/libbridge.h
@@ -20,6 +20,7 @@
 #define _LIBBRIDGE_H
 
 #include <sys/socket.h>
+#include <netinet/in.h>
 #include <linux/if.h>
 #include <linux/if_bridge.h>
 
